.eslintrc.js

```diff
@@ -0,0 +1,60 @@
+module.exports = {
+  env: {
+    browser: true,
+    es6: true,
+  },
+  extends: 'airbnb-base',
+  globals: {
+    Atomics: 'readonly',
+    SharedArrayBuffer: 'readonly',
+  },
+  parserOptions: {
+    ecmaVersion: 2018,
+    sourceType: 'module',
+  },
+  rules: {
```

Comment:
Do we really need to configure any rules here? @ooyala-amohnatkin

Reply:
There are a lot of errors fixed automatically and we need to merge it ASAP to avoid conflicts in the future. Other errors will be fixed in another pr